namespaces/docker.yml

```diff
@@ -9,3 +9,103 @@ namespace:
     type: string
     description: >
       Docker container id
+  - name: event
+    type: group
+    description: >
+      Event emitted by docker daemon that describes some activity that occurred in the past
+    fields:
+    - name: action
+      type: string
+      example: create
+      description: >
+        What activity occurred
+    - name: status
+      type: string
+      example: create
```

Comment:
Yes, the verb is very often the same for `status` and `action`
Yes, I'll add some example events there.